ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyOptimizer.cs

```diff
@@ -84,7 +84,8 @@ public class NativeViewHierarchyOptimizer
                     themedContext,
                     node.ReactTag,
                     node.ViewClass,
-                    initialProperties);
+                    initialProperties,
+                    node.RootNode.ReactTag);
```

Comment:
>node.RootNode.ReactTag [](start = 20, length = 22)

Can't we just pass the rootTag through here from UIManagerModule?

Reply:
Ok, I could do it, but there's another case of _uiViewOperationQueue.EnqueueCreateView (inside TransitionLayoutOnlyViewToNativeView) that would still have to peek into the node though.. It would add a style inconsistency.
Do you care strongly about this comment? :)

---
In reply to: [166570570](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r166570570) [](ancestors = 166570570)

Reply:
I don't care *strongly*, and I also noticed (and double checked) whether we could do this below. I don't care either way, it's not like the tag has to be computed.

---
In reply to: [166669722](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r166669722) [](ancestors = 166669722,166570570)